app/resources/add_note_base.html.template

```diff
@@ -104,103 +89,103 @@
         <div class="end-multi-columns"></div>
       </div>
 
-      <a href="#" class="status-button" style="text-decoration:none">
-        {% trans "+ Other status of person detail" %} <br>
-        <span class=hint>
-        {% trans "(Last known location/ other photos)" %}
-        </span>    
-      </a>
+        <a href="#" class="expand-button">
+          {% trans "+ More information about status of the person" %} <br>
+          <span class="hint">
+          {% trans "(Last known location / other photos)" %}
```

Comment:
for indent one more, do you mean indent the content in the span tag or indent the whole a tag?

is this okay? 

```
            </div>          

              <a href="#" class="expand-button">
                {% trans "+ More information about source of record" %} <br>
                <span class="hint">
                  {% trans "(Your phone number / Expiry)" %}
                </span>
              </a> 
```